src/theme/ItaliaTheme/Subsites/ItaliaTheme/Blocks/_highlitedContent.scss

```diff
@@ -1,13 +1,104 @@
 @use '../../all_variables' as *;
 
-@mixin block($subsite-link-color) {
+@mixin block(
+  $subsite-link-color,
+  $subsite-primary-text,
+  $subsite-secondary,
+  $subsite-secondary-text
+) {
   .block.highlitedContent {
-    .card-title {
-      a {
-        &:hover,
-        &:focus {
-          color: $subsite-link-color;
+    .bg-primary {
+      .card .card-body {
+        .categoryicon-top {
+          .text {
+            color: $subsite-primary-text;
+          }
+
+          .icon {
+            color: $subsite-primary-text;
+            fill: $subsite-primary-text;
+          }
+        }
+
+        .card-title a {
+          color: $subsite-primary-text !important;
+        }
+
+        .card-text {
+          color: $subsite-primary-text;
+        }
+
+        .read-more {
+          color: $subsite-primary-text;
+
+          .icon {
+            fill: $subsite-primary-text;
+          }
+        }
+
+        .chip.chip-primary {
+          border-color: $subsite-link-color;
+
+          & > .chip-label {
+            color: $subsite-link-color;
+          }
+
+          &:hover,
+          &:active {
+            > .chip-label {
+              color: $subsite-primary-text;
+            }
+          }
+        }
+      }
+    }
+
+    .bg-secondary {
+      .card .card-body {
+        .categoryicon-top {
+          .text {
+            color: $subsite-secondary-text;
+          }
+
+          .icon {
+            color: $subsite-secondary-text;
+            fill: $subsite-secondary-text;
+          }
+        }
+
+        .card-title a {
+          color: $subsite-secondary-text !important;
```

Comment:
È proprio obbligatorio avere questo `!important`? Occhio anche che important in questi casi di solito vince anche su `:hover` e `:focus` quindi di solito tocca cambiare anche quelli quando si cambia il colore del link

Reply:
Ho dovuto fixare un po' di cose in giro, ma corretto